nautobot/users/tests/test_views.py

```diff
@@ -169,24 +169,27 @@ def get_view_url_for_saved_view(self, saved_view, action="detail"):
     def test_get_object_anonymous(self):
         # Make the request as an unauthenticated user
         self.client.logout()
-        response = self.client.get(self._get_queryset().first().get_absolute_url())
+        response = self.client.get(self._get_queryset().first().get_absolute_url(), follow=True)
         self.assertHttpStatus(response, 200)
         response_body = response.content.decode(response.charset)
         self.assertIn(
-            "/login/?next=" + self._get_queryset().first().get_absolute_url(), response_body, msg=response_body
+            f'<input type="hidden" name="next" value="{self._get_queryset().first().get_absolute_url()}" />',
+            response_body,
+            msg=response_body,
         )
```

Comment:
what's the story on this change? Retrieving a saved view should still require `view` permissions on the object list view that it's associated with, right? So an unauthenticated user should still get redirected to login?

Reply:
This should be 
```
url = reverse(instance.view)
self.assertIn("/login/?next=" + url, response_body, msg=response_body)
```

Reply:
Sorry it should be `self.assertRedirects(response, f"/login/?next={instance.get_absolute_url()}")`